src/main/java/com/uid2/shared/attest/UidCoreClient.java

```diff
@@ -110,6 +117,7 @@ private InputStream getWithAttest(String url) throws IOException, UidCoreClientE
     private boolean attestIfRequired(HttpURLConnection conn) throws IOException, UidCoreClientException {
         boolean attested = false;
         int statusCode = conn.getResponseCode();
+        notifyResponseStatusWatcher(statusCode);
```

Comment:
What is the reason for notifying response status here?

Reply:
All response codes are checked, not just attest requests. A malicious party could block core request on purpose.